zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1962,8 +1962,10 @@ protected void doRetry() {
       });
     } catch (RuntimeException e) {
```

Comment:
nit, I slightly prefer the following alternative.

} catch (ZkSessionMismatchedException e) {
  ...
} catch (RuntimeException e) {
  ...
}

Reply:
Thanks, it is easier to read the way you suggested and I fixed it.